hal/network/lwip/esp32/esp32ncpnetif.cpp

```diff
@@ -204,28 +223,21 @@ void Esp32NcpNetif::loop(void* arg) {
 
 int Esp32NcpNetif::up() {
     NetifEvent ev = NetifEvent::Up;
-    int r = os_queue_put(queue_, &ev, 0, nullptr);
-    if (r) {
-        // FIXME: we'll miss this event, just as an attempt to potentially recover
-        // set internal state.
-        up_ = true;
-    }
-    return r;
+    return os_queue_put(queue_, &ev, 0, nullptr);
```

Comment:
Again, we've just removed the workaround for cases where the queue is filled up and the event will be completely missed.